middlewares/auth.middleware.js

```diff
@@ -1,19 +1,30 @@
 "use strict";
 const jwt = require("jsonwebtoken");
 
-const authMiddleware = async (req, res, next) => {
-
+exports.guard = async (req, res, next) => {
```

Comment:
the guard name doesn't define what it is doing. try to give a name with authorization